run.py

```diff
@@ -3,7 +3,7 @@
 from config import app_config
 from flask_jwt_extended import JWTManager
 
-config_name = os.getenv('APP_SETTINGS')
+config_name = 'testing'#os.getenv('APP_SETTINGS')
```

Comment:
why are you defining the config_name variable as `testing`, the `os.getenv('APP_SETTINGS')` should work

Reply:
This was temporary I forgot to undo..